<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>QuestionWidgetBase</class>
 <widget class="QWidget" name="QuestionWidgetBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Quizz</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" colspan="2" >
      <widget class="QProgressBar" name="progressQuestion" >
       <property name="value" >
        <number>0</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QPushButton" name="pbnQuiz" >
       <property name="text" >
        <string>Quiz Me!</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLineEdit" name="leAnswer" />
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="lblQuestion" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>leAnswer</sender>
   <signal>returnPressed()</signal>
   <receiver>pbnQuiz</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>205</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel" >
     <x>321</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
